lib/scrabble.rb

```diff
@@ -0,0 +1,60 @@
+# make a module
+class Scrabble
+
+ 	POINTS = {
+	  	"a" => 1, "b" => 3, "c" => 3, "d" => 2, "e" => 1, "f" => 4,
+	  	"g" => 2, "h" => 4, "i" => 1, "j" => 8, "k" => 5, "l" => 1,
+	  	"m" => 3, "n" => 1, "o" => 1, "p" => 3, "q" => 10, "r" => 1,
+	  	"s" => 1, "t" => 1, "u" => 1, "v" => 4, "w" => 4, "x" => 8,
+	  	"y" => 4, "z" => 10
+  	}
+
+  	VALID_LETTERS = POINTS.keys.reduce(:+)
+
+  	def initialize
```

Comment:
Feel free to exclude this method since you aren't specifying any additional logic here